pallets/runtime/tests/identity_test.rs

```diff
@@ -81,31 +98,31 @@ fn add_claims_batch() {
             },
         ];
         assert_ok!(Identity::add_claims_batch(
-            Origin::signed(claim_issuer.clone()),
+            Origin::signed(cdd_claim_issuer),
             claim_records,
         ));
 
         let claim1 = Identity::fetch_claim(
```

Comment:
The error is on this line actually.

Reply:
Because I didn't clone the value ?

Reply:
No, because you supply `cdd_claim_id` which is not present in `claim_records`. BTW we name such IDs `did` by convention, so it can be `cdd_claim_did` for example.

Reply:
ah okay I will make the changes by adding `cdd_claim_did` as the `target` in the batch claims and changing the `fetch_claim` to match the change as well.

Reply:
I made the changes but the error persists I was working on it over the weekend but no solution as yet